front-end/src/app/home/home.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, OnInit, EventEmitter, Input, Output } from '@angular/core';
+import { timer } from 'rxjs';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.scss'],
+})
+export class HomeComponent implements OnInit {
+  @Output() finished = new EventEmitter<void>();
+
+  constructor() { }
+
+  ngOnInit() {
+    timer(1000).subscribe(()=>(this.home_popup = true))
+  }
+
+  home_popup : boolean = false;
+  login_popup : boolean = false;
+  register_popup : boolean = false;
+  why_popup : boolean = false;
```

Comment:
This works for now but it allows the possibility of two of those being true. I would change the logic to use an enum variable called popup that can be one of the 4 popups.